### PR TITLE
5182 - Follow up Fix for Toolbar flex missing x icon mobile (Classic Theme View)

### PR DESCRIPTION
**Pull request by tjamesallen15**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

This PR fixes the close button not appearing in classic theme view. This is a follow up fix for the immediate found bug.

**Related github/jira issue (required)**:
Closes https://github.com/infor-design/enterprise/issues/5182

**Steps necessary to review your pull request (required)**:
- Pull this branch, build, and run the app
- Go to http://localhost:4000/components/toolbar-flex/example-index.html
- Switch to mobile view or Open in Browserstack mobile device
- Click searchfield and enter some text

**Included in this Pull Request**:
~~- [ ] An e2e or functional test for the bug or feature.~~
~~- [] A note to the change log.~~

### CODE REVIEW

File: src/components/searchfield/_searchfield-toolbar.scss [hunk 1]
```diff
@@ -1028,3 +1028,16 @@ html[class*='theme-new-'] .header > .toolbar > .buttonset > .toolbar-searchfield
     top: 1px !important;
   }
 }
+
+html[class*='theme-classic-'] {
```
Comment (tmcconechy) [inline src/components/searchfield/_searchfield-toolbar.scss:1032]: Wondering if we really want to isolate this to the one theme?

Comment (ericangeles) [reply src/components/searchfield/_searchfield-toolbar.scss:1032]: @tmcconechy agreed with you. If the issue is occurring on classic, I would suggest to use the class scss file, and if the fix affects the new, then fix the rule on _{component}-new.scss file. It defeats the purpose of the two scss file.

**Comment by tmcconechy** [comment]
So just a wierd thing im seeing in this video https://drive.google.com/file/d/1vFc_xCGUH3WE00mc6oNrR1j1jYjNdKyu/view
If you click in each field the search field "moves"

**Review by EdwardCoyle — COMMENTED** [review]
Aside from @tmcconechy's comment I think this looks good, will approve after that one 👍

**Comment by tmcconechy** [comment]
Seems like either this broke some of these examples https://github.com/infor-design/enterprise/pull/5300/checks?check_run_id=2818277375#step:11:2958 or they might need image updates.

You can check the pages in these examples and see if they look aligned right if they do we would need new images. https://github.com/infor-design/enterprise/blob/main/docs/TESTING.md#working-with-visual-regression-tests

**Comment by tjamesallen15** [comment]
@tmcconechy  i checked the other images that was affected and I will fix some of it. However, for the other ones I will push an updated baseline images

**Comment by ericangeles** [comment]
@tjamesallen15 i fixed the merging issue. The only issue now here is the linter.

**Comment by tjamesallen15** [comment]
Thanks @ericangeles . I'll update this lint issue. :)

**Comment by tmcconechy** [comment]
Looks like we might have to update a few images https://github.com/infor-design/enterprise/pull/5300/checks?check_run_id=2839139153#step:11:2948 not sure if this is cosnidered ready yet to review?

**Comment by tjamesallen15** [comment]
> Looks like we might have to update a few images https://github.com/infor-design/enterprise/pull/5300/checks?check_run_id=2839139153#step:11:2948 not sure if this is cosnidered ready yet to review?

Hello @tmcconechy . My bad. I'm checking why the baseline images are not updated when I pushed it. I did follow the procedure but I think I experience some caching issues with my docker. @ericangeles  help me on what to do with caching issue.

**Review by tmcconechy — APPROVED** [review]
LGTM 👍🏻

**Review by ericangeles — APPROVED** [review]

**Pull request merged**